.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,31 @@
-### Motivation
-
-- [Related story](http://...)
-
-TODO: write a motivation.
-
-### Test plan
-
-- TODO: Write a test plan.
-
-### Pre-merge checklist
-
-- [ ] Documentation
-- [ ] Unit tests
-- [ ] Reviews
-  - [ ] Code review
-  - [ ] Design review
-- [ ] Manual testing
-  - [ ] Windows 7 IE 11
-  - [ ] Windows 10 Edge
-  - [ ] Mac OS El Capitan Safari
-  - [ ] Mac OS El Capitan Chrome
-  - [ ] Mac OS El Capitan Firefox
-  - [ ] iOS 9 Safari
-  - [ ] Android 6 Chrome
-  - [ ] Listen to the site on a screen reader
-  - [ ] Navigate the site using the keyboard only
-- [ ] Tester approved
+_(delete unused sections)_
+
+### Changes
+
+#### Trello Ticket / Github Issue Link
+
+#### Description
+
+#### Screenshots (if applicable)
+
+### Test
+
+#### Areas of risk, including severity per area (low :blush:, medium :grimacing:, high :scream:)
```

Comment:
What do you mean by areas of risk?

Reply:
Which part of the site is affected eg. blog, particular slice, navigation etc